### PR TITLE
[docs] Add sudo-chown example & fail-fast remark

### LINKED ISSUE #29: Better doc for permission-mismatch workaround when using custom_script or so
href: https://coq.zulipchat.com/#narrow/stream/237977-Coq-users/topic/GitHub.20actions.20CI.20with.20make.20rather.20than.20opam/near/217079675

### PR DESCRIPTION
**Pull request by erikmd**
* **Kind:** docs
* Close #29

### CODE REVIEW

**Comment by erikmd** [comment]
@Zimmi48 this PR looks ready; here is the direct link to the (temporary) branch's README:
https://github.com/coq-community/docker-coq-action/tree/update-docs#readme

File: README.md [hunk 1]
```diff
@@ -130,7 +134,37 @@ This is not an issue when relying on `opam` to build the Coq project.
...
+        ...
+        startGroup Workaround permission issue
+          sudo chown -R coq:coq .  # <--
+        endGroup
+        startGroup Run tests
+          make -C tests
+        endGroup
+  - name: Revert permissions
+    # to avoid a warning at cleanup time
+    if: ${{ always() }}
```
Comment (Zimmi48) [inline README.md:161]: Is this line useful?

**Review by Zimmi48 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (erikmd) [reply README.md:161]: I guess so: it should ensure that the final `chown` is run even if the previous (docker-coq-action) step failed

Comment (Zimmi48) [reply README.md:161]: Sounds good.

**Comment by Zimmi48** [comment]
Don't you want to take care of merging the PRs yourself when you think they are ready?

**Comment by erikmd** [comment]
> Don't you want to take care of merging the PRs yourself when you think they are ready?

OK @Zimmi48, I can do this tomorrow for instance, when you'll have formally approved the PRs :)

And thanks for your comments!

**Comment by erikmd** [comment]
Merging that docs PR now

**Pull request merged**